docs/getting-started-guides/clc.md

```diff
@@ -7,12 +7,23 @@ These scripts handle the creation, deletion and expansion of kubernetes clusters
 
 You can accomplish all these tasks with a single command. We have made the Ansible playbooks used to perform these tasks available [here](https://github.com/CenturyLinkCloud/adm-kubernetes-on-clc/blob/master/ansible/README.md).
 
+## Change History
+
+The v0.5 release included the following major changes:
+
+- Kubernetes 1.2
+- Integrated Load Balancing, i.e. Kubernetes services of type LoadBalancer now automatically get a public IP address routed to all minions in the cluster via the CenturyLink Cloud LB service.
+
+For a detailed change history, please visit [the CenturyLink Cloud release notes page](https://github.com/CenturyLinkCloud/adm-kubernetes-on-clc/releases).
+
 ## Find Help
 
 If you run into any problems or want help with anything, we are here to help. Reach out to use via any of the following ways:
+
 - Submit a github issue
```

Comment:
You should probably have a link to what github repo you want issues on. By default they may assume the kubernetes docs repo.